tests/helpers.py

```diff
@@ -124,6 +124,25 @@ def send_create_update_tx(name, ddo, flags, account):
     return txn_receipt, dt_contract, erc20_address
 
 
+def send_set_metadata_state_tx(ddo, account):
+
+    datatoken_address = ddo["dataToken"]
+
+    web3 = get_web3()
+    web3.eth.default_account = account.address
+
+    dt_contract = get_web3().eth.contract(
+        abi=ERC721Template.abi, address=datatoken_address
+    )
+
+    txn_hash = dt_contract.functions.setMetaDataState(
+        2,
+    ).transact()
+    txn_receipt = get_web3().eth.wait_for_transaction_receipt(txn_hash)
+
+    return txn_receipt
```

Comment:
please also add a test (or more lines to this one), where the state is set back to 0. Assert that the asset is missing from ES after sending 2, then assert it's back after sending 0.